# SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR)
# SPDX-License-Identifier: Apache-2.0

bool s0
float32 s1
float64 s2 
int8 s3
uint8 s4
---
int16 i_16
uint16 u_16
int32 i_32
uint32 u_32
int64 i_64
uint64 u_64